### PR TITLE
Minor Change to Ascended Goat Plushie to make event run smoother.

### PR DESCRIPTION
**Pull request by Xoxeyos**
Gives Brutus three seconds to realize that he's messed up, before he's gibbed.

:cl:  
tweak: There is a 3 second delay before you are gibbed when running this certain event.  
/:cl:

### CODE REVIEW

**Comment by ynot01** [comment]
sleep() bad

File: yogstation/code/game/objects/items/plushes.dm [hunk 1]
```diff
@@ -116,6 +116,7 @@
 		user.visible_message("<span class='notice'>[user] attempts to stab [src]!</span>", "<span class='suicide'>The knife bounces off [src]'s back before breaking into millions of pieces... [src] glares at [user]!</span>") // You fucked up now son
 		I.play_tool_sound(src)
 		qdel(I)
+		sleep(3 SECONDS)
```
Comment (redmoogle) [inline yogstation/code/game/objects/items/plushes.dm]:
```suggestion
		addtimer(CALLBACK(user, /mob/living/.proc/gib), 3 SECONDS) 
```
Get rid of user.gib() and sleep. those are bad

**Review by redmoogle — CHANGES_REQUESTED** [review]

File: yogstation/code/game/objects/items/plushes.dm (hunk 1 above)
Comment (Xoxeyos) [reply yogstation/code/game/objects/items/plushes.dm]: Sure, I can do that.

**Review by redmoogle — APPROVED** [review]
Epic, it's good to go

**Comment by Xoxeyos** [comment]
Alright, I'm finished here.

**Pull request merged**